modules/bday.js

```diff
@@ -99,42 +138,102 @@ const worker = async ({ command, channel_id, channel_name, response_url }) => {
   // send card to bday user
   if (command.includes('send') && command.includes('to')) {
     const R = /<(?<MIRO_URL>.*)> to <@(?<BDAY_USER_ID>.*)\|(?<BDAY_USER_NAME>.*)>/
+    const RwithMessage = /<(?<MIRO_URL>.*)> to <@(?<BDAY_USER_ID>.*)\|(?<BDAY_USER_NAME>.*)> (?<CUSTOM_MESSAGE>.*)/
     const matches = command.match(R)
+    const matchesWithOptMessage = command.match(RwithMessage)
+
     // notify user of invalid input
     if (!matches) {
       return invalidInputNotice(response_url, command)
     }
     // send to bday user
     const { groups: { MIRO_URL, BDAY_USER_ID, BDAY_USER_NAME } } = matches
+    const sender = await web.users.info({ user: user_id })
+    const bdayPerson = await web.users.info({ user: BDAY_USER_ID })
+    const bdayPersonFullName = bdayPerson.user.real_name
```

Comment:
can destructure here I guess?
```
// not sure if real_name or user, gotta check the response
const { real_name } = await web.users.info({ user: BDAY_USER_ID })
```

Reply:
also this can be done only once out of the `if`'s cause they are the same right?

Reply:
oooh yes totally for destructuring. not for the putting outside of the IF since I'm getting the BDAY_USER_ID based on the regex expression which varies

Reply:
I'll change this in the parent PR for this